config/application.rb

```diff
@@ -43,5 +43,7 @@ class Application < Rails::Application
     config.active_job.queue_adapter = :sidekiq
 
     config.session_store :active_record_store, key: "_register_trainee_teachers_session"
+
+    config.i18n.load_path += Dir[Rails.root.join("config/locales/**/*.{rb,yml}")]
```

Comment:
What's with the `*.rb`?

Reply:
Good point, we shouldn't need it, will change.